app/src/main/java/com/google/droidjump/models/GameItem.java

```diff
@@ -14,25 +14,21 @@
  * limitations under the License.
  */
 
-package com.google.droidjump;
+package com.google.droidjump.models;
 
+import android.content.res.Resources;
 import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
 
-public class DrawableElement {
-
+abstract class GameItem {
```

Comment:
why is this class abstract?

Reply:
Because we shouldn't be able to create an object of this class. I think so.

Reply:
okay. Thanks :)